backend/src/main/java/com/funeat/review/application/SortReviewService.java

```diff
@@ -0,0 +1,157 @@
+package com.funeat.review.application;
+
+import static com.funeat.review.exception.ReviewErrorCode.REVIEW_NOT_FOUND;
+import static com.funeat.review.exception.ReviewErrorCode.REVIEW_SORTING_OPTION_NOT_FOUND;
+
+import com.funeat.member.domain.Member;
+import com.funeat.product.domain.Product;
+import com.funeat.review.domain.Review;
+import com.funeat.review.dto.SortingReviewDto;
+import com.funeat.review.exception.ReviewException.ReviewNotFoundException;
+import com.funeat.review.exception.ReviewException.ReviewSortingOptionNotFoundException;
+import com.funeat.review.persistence.ReviewRepository;
+import com.funeat.tag.persistence.TagRepository;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SortReviewService {
+
+    private static final Long FIRST = 0L;
+    private static final int TEN = 10;
+    private static final int ELEVEN = 11;
+    private static final PageRequest PAGE = PageRequest.ofSize(ELEVEN);
+    private static final String FAVORITE_COUNT_DESC = "favoriteCount,desc";
+    private static final String CREATED_AT_DESC = "createdAt,desc";
+    private static final String RATING_ASC = "rating,asc";
+    private static final String RATING_DESC = "rating,desc";
+
+    private final ReviewRepository reviewRepository;
+    private final TagRepository tagRepository;
+
+
+    public SortReviewService(final ReviewRepository reviewRepository, final TagRepository tagRepository) {
+        this.reviewRepository = reviewRepository;
+        this.tagRepository = tagRepository;
+    }
+
+    public List<SortingReviewDto> execute(final Product product, final Member member,
+                                          final Long lastReviewId, final String sort) {
+        final List<SortingReviewDto> sortingReviewsWithoutTags = getSortingReviews(product, member, lastReviewId, sort);
+        final List<SortingReviewDto> sortingReviews = addTagsToSortingReviews(sortingReviewsWithoutTags);
+
+        return sortingReviews;
```

Comment:
```suggestion
        return addTagsToSortingReviews(sortingReviewsWithoutTags);
```
Inline으로 가져가도 될 것 같아요!

Reply:
파일 삭제 했어요~